src/app/shared/form-controls/material-selection-list/material-selection-list.component.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { MaterialSelectionListComponent } from "./material-selection-list.component";
+import { FormControlBase } from "../form-contol-base.model";
+import { MaterialModule } from "src/app/shared/material/material.module";
+import {
+  FormControl,
+  FormGroup,
+  FormsModule,
+  ReactiveFormsModule
+} from "@angular/forms";
+import { By } from "@angular/platform-browser";
+
+describe("MaterialSelectionListComponent", () => {
+  let component: MaterialSelectionListComponent;
+  let fixture: ComponentFixture<MaterialSelectionListComponent>;
+  let controlProperties: FormControlBase = {
+    key: "initialKey",
+    label: "initialLabel",
+    options: [
+      { key: "initialOptionKey1", value: "initial option value 1" },
+      { key: "initialOptionKey2", value: "initial option value 2" }
+    ],
+    order: 1,
+    controlType: "selectionList",
+    initialValue: []
+  };
+  let data: any = {};
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      imports: [MaterialModule, FormsModule, ReactiveFormsModule],
+      declarations: [
+        MaterialSelectionListComponent,
+        MaterialSelectionListComponent
```

Comment:
`MaterialSelectionListComponent` is declared twice.

Reply:
Thanks. Fixed